### PR TITLE
JBIDE-24306 - remove outdated code that disables host in server editor

### PR DESCRIPTION
**Pull request by robstryker**
# Pull Request Checklist
## General
* Is this a blocking issue or new feature? If yes, QE needs to +1 this PR

## Code
* Are method-/class-/variable-names meaningful?
* Are methods concise, not too long?
* Are catch blocks catching precise Exceptions only (no catch all)?

## Testing
* Are there unit-tests?
* Are there integration tests (or at least a jira to tackle these)?
* Is the non-happy path working, too?
* Are other parts that use the same component still working fine?

## Function
* Does it work?

### CODE REVIEW

**Comment by apupier** [comment]
I created https://github.com/jbosstools/jbosstools-openshift/pull/1482
Might be good to use the 2 approcahes

**Comment by robstryker** [comment]
testPR

**Comment by robstryker** [comment]
testPR

File: plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/server/OpenShiftServerEditorSection.java [hunk 1]
```diff
@@ -616,7 +594,7 @@ private Job loadingProjectsJob(OpenShiftServerEditorModel model, IServerWorkingC
 	private void loadResources(final Composite container, OpenShiftServerEditorModel model) {
 		IServerWorkingCopy server = input.getServer();
 
-		final Connection connection = OpenShiftServerUtils.getConnection(server);
+		OpenShiftServerUtils.getConnection(server);
```
Comment (adietish) [inline plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/server/OpenShiftServerEditorSection.java]: why is this operation even required if the result isnt used? If there's a reason for it we should add least comment it.

Comment (apupier) [inline plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/server/OpenShiftServerEditorSection.java]: I don't know if this method has a side effect or not. So I kept it, I cleaned removed the unused parameter but didn't had time to check if the getConnection call has side effect (creating the connection if not yet available for instance)

Comment (robstryker) [inline plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/server/OpenShiftServerEditorSection.java]: I can't see any side-effect at all. Seems to be a 100% static getter. Doesn't seem necessary at all. Removing it.

**Comment by robstryker** [comment]
pushed to master and maintenance

**Pull request closed without merging**